src/businessrules/order/usecases/CreateOrderUseCase.java

```diff
@@ -0,0 +1,67 @@
+package businessrules.order.usecases;
+
+import businessrules.dai.AddonRepository;
+import businessrules.dai.CustomerRepository;
+import businessrules.dai.FoodRepository;
+import businessrules.dai.OrderRepository;
+import businessrules.loaders.CustomerLoader;
+import businessrules.loaders.OrderLoader;
+import businessrules.loaders.SingletonLoader;
+import businessrules.order.inputboundaries.CreateOrderInputBoundary;
+import businessrules.outputboundary.OrderModel;
+import entities.Customer;
+import entities.Order;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class CreateOrderUseCase implements CreateOrderInputBoundary {
+    OrderModel orderView;
```

Comment:
for sake of convention/consistency should we call this attribute orderModel instead?